src/main/ts/utils/ObjUtils.ts

```diff
@@ -9,9 +9,18 @@ export const lookup = <T>(o: T, k: keyof T): Option<T[keyof T]> =>
   hasKey(o, k) ? O.some(o[k]) : O.none;
 
 export const map = <A, B>(o: Record<string, A>, f: (a: A) => B): Record<string, B> => {
-  const r: Record<string, B> = {};
+  const r = {} as Record<string, B>;
```

Comment:
Isn't the previous stye kinda better? That way if someone changes the code in a way that it's no longer a valid assignment the cast won't silently work?

Reply:
Yeah, I think the old style is better - a type judgement, rather than a cast. However, it doesn't work for the `fromPairs` function below... for reasons I don't understand. I only changed `map` to be consistent with it, but wasn't sure if this was the right thing to do.